app/src/main/java/com/example/practica2/AttendanceActivity.kt

```diff
@@ -1,11 +1,119 @@
 package com.example.practica2
 
-import androidx.appcompat.app.AppCompatActivity
+import CalendarAdapter
+import CalendarItem
+import android.content.Intent
 import android.os.Bundle
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.practica2.data.Student
+import com.example.practica2.databinding.ActivityAttendanceBinding
+import com.google.android.material.appbar.AppBarLayout
+import java.time.DateTimeException
+import java.time.LocalDate
+import java.time.Month
 
 class AttendanceActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityAttendanceBinding
+    private var isTableMode = false
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_attendance)
+        binding = ActivityAttendanceBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
+
+        val backArrow = binding.backArrow
+        val personImage = binding.personImage
+        val nameLabel = binding.nameLabel
+        val appBarLayout = binding.appBar
+        val changeViewButton = binding.changeViewButton
```

Comment:
[CDL = Cerrar Depués de Leer = No hace falta que lo cambies, sólo lee (y entiende) el comentario]

Lo puedes envolver en un `with` para evitar repetir binding.

```suggestion
with (binding) {
        val changeViewButton = changeViewButton
```